docs/concepts/consensus/comparing-poa.md

```diff
@@ -68,11 +70,11 @@ Networks with three or fewer validators can produce blocks but do not guarantee
 
 !!! important
 
-    We do not recommend using IBFT and QBFT networks with three nodes for production purposes.
+    We don't recommend using QBFT and IBFT networks with three nodes for production purposes.
```

Comment:
Nit: It's an existing issue, but would it be better to say, "We recommend using QBFT and IBFT with at least four nodes in production environments."